helm_deploy/apply-for-legal-aid-uat/values-uat.yaml

```diff
@@ -0,0 +1,98 @@
+# Default values for apply-for-legal-aid-uat.
```

Comment:
This file doesn't seem to be encrypted :cry:

Reply:
That's annoying, it was showing as encrypted locally. I've force-pushed the encrypted version now. I think that's resolved it.